Part2_Mapplots/ex2_4_colorbars.ipynb

```diff
@@ -1,20 +1,5 @@
 {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    h = pr.hist.plot(
</pre><p>Why did we change to plotting hist instead of the pcolormesh?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/C2SM/pyvis/pull/95/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>I tried to add some more uses of the xarray interface. So it calls <code>pcolormesh</code>. I find it important to show/ learn the pyplot interface but I almost always use the xarray interface in practice. I am not sure how to best communicate this. </p>